tests/sentry/incidents/action_handlers/__init__.py

```diff
@@ -0,0 +1,21 @@
+import abc
+
+from sentry.incidents.logic import update_incident_status
+from sentry.incidents.models import Incident, IncidentStatus, IncidentStatusMethod
+
+
+class FireTest:
+    @abc.abstractmethod
```

Comment:
I think you need to use an abc for this to work, otherwise just raising `NotImplementedError` is also fine.